pt-BR/active_record_querying.md

```diff
@@ -1802,18 +1808,19 @@ Client.select(:id).map(&:id)
 Client.select(:id, :name).map { |c| [c.id, c.name] }
 ```
 
-with:
+por:
 
 ```ruby
 Client.pluck(:id)
 # or
 Client.pluck(:id, :name)
 ```
 
-Unlike `select`, `pluck` directly converts a database result into a Ruby `Array`,
-without constructing `ActiveRecord` objects. This can mean better performance for
-a large or often-running query. However, any model method overrides will
-not be available. For example:
+
+Ao contrário do `select`, o `pluck`  converte diretamente um resultado do banco de dados para uma lista Ruby, sem construir objetos `ActiveRecord`.
+Isso pode significar melhor desempenho para uma consulta grande ou frequente.
+No entanto, qualquer método no model que sobrescreverá os padrões não estará disponível
```

Comment:
```suggestion
No entanto, qualquer método no _model_ que sobrescreverá os padrões não estará disponível
```

Aqui é o mesmo caso que o @joaorsalmeida comentou acima. Estamos tentando manter estrangeirismos mesmo termos técnicos quando não está diretamente ligado a código em itálico. Pode dar uma olhada?